uiautomationutilities/perf_tool/readme.md

```diff
@@ -8,148 +8,106 @@
 
```

Comment:
Are there written step-by-step instructions for setting up the C# perf tool project? Is that forthcoming?

Reply:
Oh, I see - it's mentioned later in this file:
https://github.com/AzureAD/microsoft-authentication-library-common-for-android/blob/e24be64132ffa78369518e1e5c8a0cf466194c18/uiautomationutilities/perf_tool/readme.md#steps-to-run-on-local-machine

Maybe add an index to this document and link to various sections from the top?

Reply:
Done